src/contexts/index.js

```diff
@@ -0,0 +1,34 @@
+import { createContext, useState } from "react";
+export const TodosListContext = createContext([]);
```

Comment:
I would rename this file to name of the context. to avoid confusion

Reply:
I would recommend setting the default value of the context to an empty object instead of  an array. since the value your setting to the context is an object not an array

Reply:
usually the default value is a fallback and it could be that if you set an incompatible types that you could potential get some errors. I would recommend checking out the best practices for this